styles/main.css

```diff
@@ -127,6 +126,135 @@ footer a:hover {
   margin-top: 25px;
 }
 
+.free-signup {
+  background-color: #f4f5f6;
+  border: 1px solid #e5e5e5;
+  width: 300px;
+}
+
+.free-signup h6 {
+  margin-left: 8px;
+  font-weight: bold;
+}
+
+.free-signup input {
+  width: 90%;
+  margin-top: 10px;
+  margin-left: 15px;
+}
+
+.free-signup .button-signup {
+  background-color: #f72210;
+  width: 90%;
+  margin: 10px auto;
+  font-weight: 600;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  font-size: 17px;
+  cursor: pointer;
+}
+
+.top-debate-div p a {
+  text-decoration: none;
```

Comment:
- [ ] [OPTIONAL] use `text-decoration-none` of the bootstrap class instead of using css rule for it.